dist/index.js

```diff
@@ -340,8 +340,14 @@ function _typeof(obj) { if (typeof Symbol === "function" && typeof Symbol.iterat
           start = index;
         }
 
+        end = start + delta.keeps - 1;
+
+        if (end > delta.total) {
+          end = delta.total - 1;
```

Comment:
It won't work properly when delta.total == 0

Reply:
Fix in version 1.4.6